lib/components/atoms/Video/types/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
+  width?: Number,
+  height?: Number,
+  autoPlay?: Boolean,
+  controls: Boolean,
+  loop?: Boolean,
+  muted?: Boolean,
+  poster?: String,
+  src: String,
+  type: String,
+  children: Node,
```

Comment:
children i believe should be optional since we have default track as fallback

Reply:
fixed